src/assets/src/components/home.tsx

```diff
@@ -30,22 +28,12 @@ function QueueLookup() {
 }
 
 export function HomePage(props: PageProps) {
-    const getUser = async () => {
-        if (!props.user) return;
-        return await apiGetUser(props.user.id)
-    }
     const [user, setUser] = useState(undefined as MyUser | undefined);
-    const [doRefreshUser, refreshUserLoading, refreshUserError] = usePromise(getUser, setUser);
-    useEffect(() => {
-        doRefreshUser();
-    }, []);
-    useAutoRefresh(doRefreshUser, 10000);
+    const userWebSocketError = useUserWebSocket(props.user!.id, (u) => setUser(u as MyUser));
```

Comment:
While I was testing this, if the user is not logged in, the home page body doesn't render because `useUserWebSocket` results in an error (there isn't a user yet). Do you get the same behavior?

Reply:
Thanks for catching that! It should be fixed if you pull latest.